tests/cc/test_cc_cqasm.py

```diff
@@ -31,42 +31,45 @@ def run_test_case(self, name):
             # ql.set_option('log_level', 'LOG_DEBUG')
             # ql.set_option('log_level', 'LOG_WARNING')
 
-            if 0:
-                # use compatibility
-                # fails in new-to-old IR conversion for CC backend
-                ql.set_option('write_qasm_files', 'yes')
-                ql.set_option('write_report_files', 'yes')
-                ql.compile(in_fn)
-
-            if 0:
-                # use compatibility
-                # fails in outputting debug info
-                ql.compile(
-                    in_fn,
-                    {
-                        # 'cqasm_file': in_fn,
-                        'output_prefix': 'test_output/%N.%P',
-                        'debug': 'yes'
-                    }
-                )
-
             if 1:
                 # use pass manager
                 pl = ql.Platform("cc", "config_cc_s17_direct_iq_openql_0_10.json")
                 c = pl.get_compiler()
 
+                # insert passes at front (in reverse run order)
```

Comment:
Could you make deadcodeelim and propagate passes that need not be configured in this way and are called by default?